testdata/go/v3/monitoring/memcached-operator/docs/monitoring/metrics.md

```diff
@@ -0,0 +1,11 @@
+# Operator Metrics
+This document aims to help users that are not familiar with metrics exposed by this operator.
```

Comment:
Thinking about it, wouldn't it be better if we integrate this with README (https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/monitoring/memcached-operator/README.md). I'm wondering if we ever need a docs/ folder? Can this not be an addition in the README file, if its going to be common throughout. 

If this can be moved to README, then the generator would just have to fill in the `Operator Metrics List` section.

Reply:
I think the metrics should be documented in a separate file, since best practice encourages generating the metrics separately from the core code.
But the README file may be more exposed and reach more users, so not sure.

Reply:
There can be a very long list of metrics and recording rules to an operator, depending on its complexity and having it in the README may result in a very long document.
But it is possible to add the link to this file from the README for better visibility.

Reply:
Can we do that in follow up?

Reply:
+1 to having a link to this doc from the README

Reply:
> Can we do that in follow up?

Yes.